master/buildbot/newsfragments/git-source-step-update.doc

```diff
@@ -0,0 +1,9 @@
+Update Git source checkout build step documentation:
+
+* remove redundant empty lines
```

Comment:
I guess the newsfragment should be reasonably concise :)

Reply:
would you like me to make it one liner?  (do we have a policy about this somewhere?)

Reply:
There is a readme file in this directory, which explains the policy. But yes, this is what is made into the changelog. So it should be concise and contain information useful for a changelog.